src/redux/books/books.js

```diff
@@ -1,44 +1,43 @@
+import axios from 'axios';
+
 const ADD_BOOK = 'book-store/books/ADD_BOOK';
 const REMOVE_BOOK = 'book-store/books/REMOVE_BOOK';
+const FETCH_BOOK = 'book-store/books/FETCH_BOOK';
+
+const baseUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/5MiZYMR1tUdUdeapIDXc/books';
 
-const initialBook = [
-  {
-    id: 1,
-    author: 'Emmanuel Mensah',
-    books: 'Fragments',
-    category: '',
-  },
-  {
-    id: 2,
-    author: 'Ama Ata Aidoo',
-    books: 'After The Ceremonie',
-    category: '',
-  },
-  {
-    id: 3,
-    author: 'Kofi Anyidoho',
-    books: 'The Word Behind Bars',
-    category: '',
-  },
-];
+const initialBook = [];
 
 const bookReducer = (state = initialBook, action) => {
   switch (action.type) {
     case ADD_BOOK: {
       const objBook = {
-        id: state.length + 1,
+        item_id: `book${Date.now()}`,
+        title: action.book.book,
         author: action.book.author,
-        books: action.book.books,
+        category: 'fiction',
       };
-      return [...state, objBook];
+      const newState = [`book${Date.now()}`, [{
+        title: action.book.book,
+        author: action.book.author,
+        category: 'fiction',
+      }]];
+
+      axios.post(baseUrl, objBook);
+      // console.log(state);
+      return [...state, newState];
     }
 
     case REMOVE_BOOK: {
-      const remove = state.filter((book) => book.id !== action.id);
+      axios.delete(`${baseUrl}/${action.id}`);
```

Comment:
- [ ] The same thing for the removeBoook action creator. I recommend adding the following code ```Axios.delete(`${baseUrl}/${action.id}`);``` to the removeBook action creator. Remember that we use action creators to send and receive books via API.